Mathlib/Analysis/BoxIntegral/UnitPartition.lean

```diff
@@ -0,0 +1,286 @@
+/-
+Copyright (c) 2024 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.Analysis.BoxIntegral.Partition.Measure
+import Mathlib.Analysis.BoxIntegral.Partition.Tagged
+
+/-!
+# Unit Partition
+
+Fix `n` a positive integer. `BoxIntegral.unitPartition.Box` are boxes in `ι → ℝ` obtained by
```

Comment:
```suggestion
Fix `n` a positive integer. `BoxIntegral.unitPartition.box` are boxes in `ι → ℝ` obtained by
```
Do you really want to use `PNat`s here? (The alternative would be to replace `(n : ℕ+)` by `(n : ℕ) [NeZero n]`.)

Reply:
Yes, it is a good idea to replace `PNat` with `NeZero`. I changed the code accordingly. Thanks for the suggestion.